### DISCUSSION TITLE
Navigation display size settings

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#129] [Q&A] [answered]
Next Cruiser will provide additional options to customize the **size** of the navigation screen.

Cruiser already provides font + display size options in "Settings | Application | Advanced",
but these options change the texts and interface of the _whole application_.

What if we _only_ want to change the sizes on the _main screen_:
- **Font**: texts in info panels)
- **Display**: direction icon in navigation)

<img src="https://user-images.githubusercontent.com/3484020/234191626-9215a3ac-18aa-4209-bcb8-7f6be2bed6ae.jpg" width="50%"/>

**Comment by DolomitesTom** [comment]
Yes, sounds good, I would prefer more granularity in size options

**Comment by devemux86** [comment]
> I would prefer more granularity in size options

Separate size options for font + display will be available in the next Cruiser update (see the image [above](https://github.com/devemux86/cruiser/discussions/129#discussion-5092896)).

**Comment by devemux86** [comment, marked as answer]
Separate size options for font + display in main screen are available in [Cruiser 3.0.18](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Available in "Settings | Application | Appearance" or search settings for "size".

---

- Better use default or 100% sizes in "Settings | Application | Advanced" that _change the whole application_
- And modify the sizes in "Settings | Application | Appearance" that _only change the main navigation screen_

**Reply by DolomitesTom** [reply]
I am already curious

**Reply by devemux86** [reply]
We could modify slightly the names of these size settings,
so users can understand that the "Appearance" settings are for the main screen
while the "Advanced" size settings change the whole application.

**Reply by DolomitesTom** [reply]
Overall Display Size vs. Navigation Panels Size

The resulted difference is great, very much like it.

**Reply by devemux86** [reply]
> Overall Display Size vs. Navigation Panels Size

Indeed, the purpose of the new size settings is to adjust the _main screen_ according to our preferences,
without having to resize the _entire application_ and potentially "break" the display of all dialogs, screens.

**Comment by devemux86** [comment]
A separate setting for the size of the screen buttons will be available in the next Cruiser:

<img src="https://github.com/devemux86/cruiser/assets/3484020/fade35c7-1cfa-4d33-bf73-e01ef19fc399" width="50%"/>

**Reply by devemux86** [reply]
A separate setting for the size of the screen buttons is available in [Cruiser 3.0.23](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

"Settings | Application | Appearance | Buttons size" or search settings for "button".

**Reply by MotoUKRider** [reply]
Thanks for this - 130% works great for me in landscape nav.

**Reply by Ciutnik** [reply]
One note: I think that since the "off-route" triangle is also a button, its size should also be variable. Today I had a problem to effectively hit it with a finger in a glove ☝️

**Reply by devemux86** [reply]
@Ciutnik good point!

This is svg, so it is easy and will be available in the next Cruiser update.

We could do the same for the compass too?
Although we need to change to svg images first.

**Reply by Ciutnik** [reply]
In my opinion, the enlarged compass is not necessarily because there is no need to press it while driving. Well, unless there is an idea to use it in the future...

**Reply by devemux86** [reply]
Currently the compass allows to change the map orientation (via long press).
[No rotation, 2D rotation, 3D rotation]

**Reply by Ciutnik** [reply]
> Currently the compass allows to change the map orientation

Yes, but these changes are not often made, especially while driving 🙂
As for me, there could be an option to switch the mode: indicating north or destination (this is how I have it in other navigation).

**Reply by devemux86** [reply]
> there could be an option to switch the mode: indicating north or destination (this is how I have it in other navigation)

@Ciutnik can you create a new topic so we can discuss it?

You could also write there more details about the "destination".
(I understand the north is the "No rotation" in Cruiser)

**Reply by devemux86** [reply]
> since the "off-route" triangle is also a button, its size should also be variable

The off route warning "button" is also changed by the "Buttons size" setting in [Cruiser 3.0.24](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by devemux86** [comment]
> what should the Appearance Display+Font sliders alter?

The size settings in "Advanced" change the whole application (there is application reference in their names).

The size settings in "Appearance" change the main screen used in navigation and follow, e.g. font and icons.
They are in the same settings area with the various color settings that change the same main screen panels.
In that area there will be a new setting for the size of the screen buttons, see image above: https://github.com/devemux86/cruiser/discussions/129#discussioncomment-6078722

As we already changed the main app menu and the various navigation action menus with new large buttons,
is there any other area that we need extra size when operating the app outside, e.g. with gloves?

**Reply by MotoUKRider** [reply]
Thanks for the "Appearance" explanation - yes, I can see that now and have adjusted them so the on-screen info is a little bigger.

**Comment by DolomitesTom** [comment]
> The size settings in "Advanced" change the whole application (there is application reference in their names).

button size is now selectable, but I still would need this advanced font size because otherwise the menu font stays small.

**Reply by devemux86** [reply]
> I still would need this advanced font size because otherwise the menu font stays small

At the very least, try changing _only the advanced font size_.

Not the advanced display size, as it changes everything in the user interface.

**Reply by devemux86** [reply]
> menu font

@DolomitesTom when you say "menu font" do you mean the menu window, for example when long press on the map?

Like the images [here](https://github.com/devemux86/cruiser/discussions/199#discussioncomment-6083034)?

**Reply by DolomitesTom** [reply]
Yes this and also the font size in menu settings including the submenus

**Comment by DolomitesTom** [comment]
> At the very least, try changing only the advanced font size.

This only changes the font size of the navigation information bars.

**Reply by devemux86** [reply]
"**Advanced | Font size (application)**" changes the size of texts in the whole application.

"**Appearance | Font size**" only changes the text in the navigation / follow screen panels.
